### ISSUE TITLE
How about the reimplementation performance compared with original paper's result?

### ISSUE THREAD

**Issue by sonack** [huaifeng1993/DFANet#1]
Great Work!
Hoping for more information about the result in readme file.

**Comment by huaifeng1993** [comment]
The training process has not been completed yet. I am waiting for the result。There are also some meticulous work to be done。i am not sure if some details of the network  was implemented correct. if you are interested in this project i think that we can cooperate to complete it.

**Comment by zhangpingz** [comment]
Hello, has the training process completed yet?

**Comment by huaifeng1993** [comment]
> Hello, has the training process completed yet?
the performance of this implementation is relatively poor. The IOU of the val set is only about 0.4, and the IOU of the training set is 0.5，with out loading imageNet pre-training model.

**Comment by zhangpingz** [comment]
I tried to train the network, but the IOU of the training set is still under 0.4.
I think it is necessary to pre-train the backbone.
And from the network, I think the loss function has been sophisticated treated, but I did not find any details in the paper.

**Comment by huaifeng1993** [comment]
@zhangpingz 增加预训练代码。但是最终结果还没有跑出来。我在imagenet上训练了22个epoch，加载第22个epoch的预训练参数。目前来看前面几个epoch提升还行。loss不是很清楚，目前只用了pytorch提供的交叉熵损失函数。

**Comment by Aktcob** [comment]
I use OHEM // random crop // multi scale. test mIoU is 51%, which is still far away from the original paper

**Comment by sonack** [comment]
> I use OHEM // random crop // multi scale. test mIoU is 51%, which is still far away from the original paper

Me too

**Comment by ngunauj** [comment]
这玩意，草参得设置好
